AnkiDroid/src/main/java/com/ichi2/libanki/template/ParsedNode.kt

```diff
@@ -39,7 +39,7 @@ abstract class ParsedNode {
     // Used only fot testing
     @VisibleForTesting
     fun template_is_empty(vararg nonempty_fields: String?): Boolean {
-        return template_is_empty(HashSet(Arrays.asList(*nonempty_fields)))
+        return template_is_empty(nonempty_fields.map { it!! }.toSet())
     }
```

Comment:
```suggestion
    fun template_is_empty(vararg nonempty_fields: String): Boolean {
        return template_is_empty(nonempty_fields.toSet())
    }
```

the callers (only tests) always send non-null args, so the param can be made non-null

Reply:
Your change is valid, but I don't think it belongs to this commit or even PR